code/sorting/quick_sort/Qick_sorting.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+ 
+using namespace std;
+ 
+void quick_sort(int[],int,int);
+int partition(int[],int,int);
+ 
+int main()
+{
+    int a[50],n,i;
```

Comment:
This makes no sense. You are setting a max size for the array. If you want to ask the user for an array size, use dynamic allocation, or better yet, vectors.